misc/formatter/modules/formatter-core/src/test/resources/declarations/class-definition/source/class_definition_declaration_2.bal

```diff
@@ -2,6 +2,8 @@ class
     Person
 
        {
+
+
```

Comment:
same here

Reply:
This is also a source file. Should it be reformatted?

Reply:
ack